### PR TITLE
Updated Kube Slack channel

### PR DESCRIPTION
**Pull request by jtligon**
updated bottom of index.md

### CODE REVIEW

**Comment by rhatdan** [comment]
@jtligon Your PR is a little hosed.  
git rebase -i origin
Save it
git push --force
Should fix it.

**Comment by jtligon** [comment]
better @rhatdan ?

**Comment by rhatdan** [comment]
LGTM

File: index.md [hunk 1]
```diff
@@ -128,7 +128,7 @@ You can find us at:
 
 - [GitHub](https://github.com/kubernetes-incubator/cri-o)
 - IRC: FreeNode, #cri-o channel
-- Slack: Kubernetes sig-node
+- Slack: Kubernetes #crio
```
Comment (runcom) [inline index.md]: I would have still left sig-node channel but cool

**Pull request merged**